archlinuxcn/devtools-archlinuxcn/PKGBUILD

```diff
@@ -15,6 +15,9 @@ optdepends=('btrfs-progs: btrfs support')
 provides=("devtools=$pkgver-$pkgrel")
 conflicts=("devtools")
 source=(https://sources.archlinux.org/other/${_pkgname}/${_pkgname}-${pkgver}.tar.gz{,.sig}
+        "01-nosetarch.patch::https://github.com/archlinux/devtools/commit/61f67fd05192a21c03a015712b0eae749ce81ccf.patch"
```

Comment:
https://github.com/archlinux/devtools/pull/43 is not merged yet, so this commit is not attached to any git reference (branch/tag), and might be removed in a future git gc on GitHub. It is better to use `https://github.com/holymonson/devtools/commit/xxx`.